Projects/Modules/NetworkModule/Sources/Interceptor/DefaultLoggingInterceptor.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import OSLog
+
+#if DEBUG
+
+public final class DefaultLoggingInterceptor: Interceptor {
+    
+    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier ?? "", category: "NETWORK")
```

Comment:
이부분 붙여주시면 좋을 것 같습니다! (코드 컨벤션)

```suggestion
public final class DefaultLoggingInterceptor: Interceptor {
    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier ?? "", category: "NETWORK")
```